opdracht4.js

```diff
@@ -0,0 +1,71 @@
+
+//     Philips 43PUS6504/12 - 4K TV
+//      €379,-
+//      43 inch (109 cm) | 50 inch (127 cm) | 58 inch (147 cm)
+// Opdracht 4a: Maak een herbruikbare functie die een string genereert
+// voor de naam van één tv en deze teruggeeft in het format [merk] [type] - [naam]
+
+function fullTvName(tv) {
+    return tv.brand + ' ' + tv.type + ' - ' + tv.name;
```

Comment:
Je kan dit nog iets aanpassen zodat het beter leesbaar wordt:
```suggestion
    return `${tv.brand} ${tv.type} - ${tv.name}`;
```